Source/Source/Trigger_SentAway.cs

```diff
@@ -7,7 +7,7 @@ public class Trigger_SentAway : Trigger
     {
         public override bool ActivateOn(Lord lord, TriggerSignal signal)
         {
-            if (signal.type == TriggerSignalType.Tick)
+            if (signal.type == TriggerSignalType.Tick && Find.TickManager.TicksAbs % 250 == 0)
```

Comment:
Hm... what's the idea behind this? Won't this cause signals to be missed?
I can imagine you'd want to prevent "visitors are leaving spam", but will this actually work?

Reply:
I don't think so, because this (used to) get executed every tick to check whether a pawn had been sent away. This at worst then will mean that the leaving of a guest group is staggered by 250 ticks, most likely laziness on my part w.r.t not adding a method to change the lords toil to leaving immediately from the send home method in `ITab_Pawn_Guest:SendHome`.

(My belief for this is because the guests .sendaway bool is only set when they are being 'cleared' for use in a lord, or when the player forces them to be sent away).

Certainly something I should look at fixing either way.

Reply:
'Fixing' may have been the wrong word. This works as is, as I mentioned prior, this at worst means that, after sending away a pawn, the player will wait 250 ticks for a guest group to be properly dismissed, restoring original functionality would be ITab_Pawn_Guest:SendHome finding the lord, and manually triggering the code which makes the lord leave. 
something like
```csharp
ITab_Pawn_Guest:SendHome ()
{
   ...
   pawn.GetLord().SendAway();
   return ...
}
```